content/4_Gold/Tree_Euler.mdx

```diff
@@ -71,6 +71,28 @@ public static void dfs(int i, int p) {
 
 </JavaSection>
 
+<PySection>
+
+```py
+g = [[] for i in range(200005)]
+st = [0 for i in range(200005)]
+en = [0 for i in range(200005)]
+pos = 1
```

Comment:
use `timer` like in the other languages of the module

Reply:
I changed it to timer